runtime/oti/jvminit.h

```diff
@@ -328,6 +328,7 @@ enum INIT_STAGE {
 #define VMOPT_XXHANDLESIGXFSZ "-XX:+HandleSIGXFSZ"
 #define VMOPT_XXHEAPDUMPONOOM "-XX:+HeapDumpOnOutOfMemoryError"
 #define VMOPT_XXNOHEAPDUMPONOOM "-XX:-HeapDumpOnOutOfMemoryError"
+#define VMOPT_XXDEBUGLOCALMAPPER "-XX:DebugLocalMapper"
```

Comment:
Should the option be -XX:+DebugLocalMapper ?

Reply:
No, a `+` version implies that there should also be a `-` version.  I've deliberately coded this so the debug local mapper can't be disabled once enabled as doing so would bad when running in debug mode